cmd/bbolt/main.go

```diff
@@ -175,6 +179,33 @@ Use "bbolt [command] -h" for more information about a command.
 `, "\n")
 }
 
+// versionCommand represents the "version" command execution.
+type versionCommand struct {
+	baseCommand
+}
+
+// newVersionCommand returns a versionCommand.
+func newVersionCommand(m *Main) *versionCommand {
```

Comment:
Please add it as a cobra style command. 

FYI. https://github.com/etcd-io/bbolt/blob/59b8453440e49926ff92da0fb06f78654e225477/cmd/bbolt/command_root.go#L20

Reply:
Ok, I will move this to cobra style command.
do you have a plan to move every other command to cobra style, I can work on that feel free to assign me in that issue

Reply:
For other commands, we are planning to migrate them to cobra style commands in 2.0. FYI. https://github.com/etcd-io/bbolt/issues/472